test/verify_enterprise_test.rb

```diff
@@ -180,7 +180,7 @@ def initialize
       assert_equal "reCAPTCHA verification failed, please try again.", @controller.flash[:recaptcha_error]
     end
 
-    it "does not verify via http call when response length exceeds G_RESPONSE_LIMIT" do
+    it "does not verify via http call when response length exceeds G_RESPONSE_MAX_LIMIT" do
```

Comment:
don't think this should change, the name references the var we send to recapcha ... possibly should just be rewritten to "exceeds limit"